scst-store/deployment-details.hbs.md

```diff
@@ -51,37 +86,15 @@ Where `PASSWORD-0123` is the same password used between deployments.
 
 >**Note:** there is a known issue related to changing database passwords [Persistent Volume Retains Data](../release-notes.md#store-persistent-volume-retains-data).
 
-### <a id='appserv-type'></a>App service type
-
-Supported values include `LoadBalancer`, `ClusterIP`, `NodePort`. The `app_service_type` is set to `LoadBalancer` by default. If your environment does not support `LoadBalancer`, and you want to use `ClusterIP`, configure the `app_service_type` property in your `metadata-store-values.yaml`:
-
-```yaml
-app_service_type: "ClusterIP"
-```
-
-If the `ingress_enabled` property is set to `"true"`, VMware recommends setting the `app_service_type` property to `"ClusterIP"`.
-
 ### <a id='service-accounts'></a>Service accounts
 
 By default, a service account with read-write privileges to the metadata store app is installed.
 This service account is a cluster-wide account that uses ClusterRole.
 If you don't want the service account and role, set the `add_default_rw_service_account` property to `"false"`.
-To create a custom service account, see [Configure access tokens](create-service-account-access-token.md).
+To create a custom service account, see [Create Service Account](create-service-account.hbs.md).
 
-The store creates a read-only cluster role, which is bound to a service account by using `ClusterRoleBinding`. To create service accounts to bind to this cluster role, see [Configure access tokens](create-service-account-access-token.md).
+The store creates a read-only cluster role, which can be bound to a service account by using `ClusterRoleBinding`. To create service accounts to bind to this cluster role, see [Create Service Account](create-service-account.hbs.md).
 
 ## <a id='export-cert'></a>Exporting certificates
 
 Supply Chain Security Tools - Store creates a [Secret Export](https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/develop/docs/secret-export.md) for exporting certificates to `Supply Chain Security Tools - Scan` to securely post scan results. These certificates are exported to the namespace where `Supply Chain Security Tools - Scan` is installed.
-
```

Comment:
There are many other deployment details/options that could be added that are not included. Do we want to add them and link them out as needed? Or continue to omit them? 

For example:
- `tls` options
- `log_level`
- configurable limit and request for cpu and memory for postgres, kube-rbac-proxy and app
- storage class

Reply:
I agree we should add those options in the documentation, I created a [story](https://jira.eng.vmware.com/browse/META-840) for that